proposals/simd/SIMD.md

```diff
@@ -784,3 +784,25 @@ Lane-wise saturating conversion from floating point to integer using the IEEE
 resulting lane is 0. If the rounded integer value of a lane is outside the
 range of the destination type, the result is saturated to the nearest
 representable integer value.
+
+### Integer to integer widening
+* `i16x8.convert_s/i8x16Low(a: v128) -> v128`
+* `i16x8.convert_s/i8x16High(a: v128) -> v128`
+* `i16x8.convert_u/i8x16Low(a: v128) -> v128`
+* `i16x8.convert_u/i8x16High(a: v128) -> v128`
+* `i32x4.convert_s/i16x8Low(a: v128) -> v128`
+* `i32x4.convert_s/i16x8High(a: v128) -> v128`
+* `i32x4.convert_u/i16x8Low(a: v128) -> v128`
+* `i32x4.convert_u/i16x8High(a: v128) -> v128`
+
+Converts low or high half of the smaller lane vector to a larger lane vector,
+with signed or unsigned saturation.
+
+### Integer to integer narrowing
+* `i8x16.convert_s/i16x8(a: v128, b: v128) -> v128`
+* `i8x16.convert_u/i16x8(a: v128, b: v128) -> v128`
+* `i16x8.convert_s/i32x4(a: v128, b: v128) -> v128`
+* `i16x8.convert_u/i32x4(a: v128, b: v128) -> v128`
```

Comment:
I'm not sure what the difference between signed and unsigned narrowing is. Can that be elaborated?

Reply:
Good question @billbudge can you help here?
I think its whether the truncation care about the sign.
E.g. given 428, 0000000110101100
a truncation unsigned will take the LSB, 10101100, and return that modulo 128, so 44 (0101100).
a truncation signed will return 10101100, which is -84.
I referred to the answer here https://stackoverflow.com/questions/19273658/integer-conversionsnarrowing-widening-undefined-behaviour

Reply:
I would have interpreted the "convert" in that SO answer as being a reinterpret that does not change any bytes. In particular LLVM does not have signed/unsigned truncation, although it does allow you to *interpret* the result as signed or unsigned after you have thrown out the most significant bits.

Reply:
The signed/unsigned in the narrowing operations refers to how these operations will handle overflow conditions. For example, in the case of i16x8.convert_s/i32x4, this will convert the 4 i32 values to 8 i16 values using signed saturation on overflow, i.e. it will map diirectly to [packssdw](https://www.felixcloutier.com/x86/packsswb:packssdw).

Reply:
Thanks for your explanation! I added a short note in 39a7408, please see if that's okay.

Reply:
Ahh, I understand now. I hadn't realized there was saturation involved :+1: